### PR TITLE
Render jobs

### LINKED ISSUE #3: Render jobs
As a user,
When I go to the root page,
I will see a list of jobs,
And each job includes the position, company, date, truncated description, and a button to view more job details.

### PR DESCRIPTION
**Pull request by ksk5280**
Created sample jobs that are rendered onto the page in a JobListings component. It currently lists all of the job attributes that are currently available from the lookingFor API. There is no styling at this point.

@rrgayhart @natevenn @brianrip 

closes #3

### CODE REVIEW

**Comment by cheljoh** [comment]
Looking good! Will work on styling once we see what the navbar looks like.

**Comment by natevenn** [comment]
@brianrip and I had issues with rendering the css styling and incorporating bootstrap but we are finally up and running. Still need to do a little more styling and break out search bar into its own component. Will have a pr for navbar after lunch!

**Comment by cheljoh** [comment]
Refactored the code since initial PR so all child elements have keys.

File: lib/index.js [hunk 1]
```diff
@@ -31,13 +31,59 @@ class Header extends React.Component {
 class JobListings extends React.Component {
   constructor() {
     super();
+    this.state = { jobs: [] };
+  }
+
+  componentWillMount() {
```
Comment (natevenn) [inline lib/index.js]: Why will mount over did mount?

File: lib/index.js [hunk 2]
```diff
@@ -34,7 +34,7 @@ class JobListings extends React.Component {
     this.state = { jobs: [] };
   }
 
-  componentWillMount() {
+  componentDidMount() {
```
Comment (natevenn) [inline lib/index.js]: looks like we are going with did mount

File: lib/index.js [hunk 3]
```diff
@@ -31,13 +31,59 @@ class Header extends React.Component {
...
+      </div>
     );
   }
 }
 
+const Technology = ({technology}) => {
+  return (<li>{technology}</li>)
+}
+
+const JobItem = ({job}) => {
```
Comment (natevenn) [inline lib/index.js]: for consistence should we change jobItem to just job like technology?

**Comment by natevenn** [comment]
I like the refactoring of job and technology classes.

**Pull request merged**